### PR TITLE
[WIP] Updates to the Technology preview tracker for Pipelines and GitOps in 4-7 Release Notes

### PR DESCRIPTION
**Pull request by Srivaralakshmi**
- **Aligned team**: Dev Tools
- **OCP version for cherry-picking**: `enterprise-4.7`
- **JIRA issues**: [RHDEVDOCS-2788](https://issues.redhat.com/browse/RHDEVDOCS-2788)
- **Preview pages**: [Technology Preview features](https://deploy-preview-31648--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-7-release-notes.html#ocp-4-7-technology-preview)
- **Reviewer**: @Preeticp @wtam2018

### CODE REVIEW

**Comment by Srivaralakshmi** [comment]
@Preeticp PTAL and provide comments. Thanks!

File: cicd/gitops/gitops-release-notes.adoc [hunk 1]
```diff
@@ -0,0 +1,21 @@
+//OpenShift GitOps Release Notes
+include::modules/gitops-document-attributes.adoc[]
```
Comment (Preeticp) [inline cicd/gitops/gitops-release-notes.adoc:2]: I am not sure why we have this file in this PR. the other PR has this file and will be merged to both 4.7 and 4.8, so we actually do not need this file in this PR. You could use `git rm file_name` to remove this from the PR.

**Comment by Preeticp** [comment]
Apart from that extra file, LGTM.

**Comment by Srivaralakshmi** [comment]
@Preeticp I have removed the gitops-document-attributes file. PTAL and let me know if there is anything else to do from my side. Thanks!

**Review by wtam2018 — APPROVED** [review]
LGTM
Thanks @Srivaralakshmi

**Pull request merged**